server/channels/app/server.go

```diff
@@ -1241,7 +1235,6 @@ func runConfigCleanupJob(s *Server) {
 }
 
 func (s *Server) runLicenseExpirationCheckJob() {
-	s.doLicenseExpirationCheck()
```

Comment:
> We also remove the first call to check license expiration which would load the license again. This is unnecessary because if the license is expired, server wouldn't start at all.

Could you point me to where the server would fail to start if license was expired on initialization?

Reply:
What I meant was that `doLicenseExpirationCheck` calls `TriggerLoadLicense` which is already called at server startup. So calling it again is useless. But I see that the license checks all happen after loading the license. I think what we can do is remove the call to `TriggerLoadLicense` and then do everything else after that. Loading the license from inside method doesn't make much sense.